### ISSUE TITLE
Request body larger than maxBodyLength limit

### ISSUE THREAD

**Issue by tiholic** [ConvertAPI/convertapi-node#16]
Error uploading a large document (24MB) for conversion using `convertapi.convert(...)`

LOGS:
```js
converting via convertapi...
Error from view function: "merge"
 { Error: Request body larger than maxBodyLength limit
    at Function.handleError (/Users/rohit/dpapp/documerge.io/doc_server/node_modules/convertapi/lib/client.js:15:262)
    at then.catch.a (/Users/rohit/dpapp/documerge.io/doc_server/node_modules/convertapi/lib/client.js:15:97)
    at processTicksAndRejections (internal/process/next_tick.js:81:5) data: null }
```

I tried adding the following and it works.
![image](https://user-images.githubusercontent.com/14252381/63927627-b811da80-ca6b-11e9-8a65-84ccd68f78cd.png)

https://github.com/ConvertAPI/convertapi-node/blob/d7700ca8cb6449117b0b5dd5a9f8962ee5805cde/src/client.js#L76

**Comment by tiholic** [comment]
@tomasr78 would it help speeding up a new release to NPM if I can give a PR?

**Comment by tomasr78** [comment]
@tiholic Our programmer @laurynas-baltsoft will look at this soon and post new release till the end of this week.

**Comment by laurynas-baltsoft** [comment]
Hey @tiholic, thanks for detailed report. Released `1.2.1` version.
